src/CommunityToolkit.Maui/Behaviors/PlatformBehaviors/StatusBar/StatusBarBehavior.shared.cs

```diff
@@ -0,0 +1,56 @@
+
+using System.Runtime.CompilerServices;
+using System.Runtime.Versioning;
+using CommunityToolkit.Maui.Core;
+using CommunityToolkit.Maui.Core.Capabilities;
+
+namespace CommunityToolkit.Maui.Behaviors;
+
+/// <summary>
+/// <see cref="PlatformBehavior{TView,TPlatformView}"/> that controls the Status bar color
+/// </summary>
+[UnsupportedOSPlatform("windows")]
+public class StatusBarBehavior : PlatformBehavior<VisualElement>
```

Comment:
Should this be `PlatformBehavior<Page>`? 

For example, I'm not sure I can think of a scenario where we would attach a `StatusBarBehavior` to a `Label`.

```suggestion
public class StatusBarBehavior : PlatformBehavior<Page>
```

Reply:
Yeah, I used the `VisualElement` to make it more general, so devs can apply the behavior everywhere. But I'm ok to make it more restrict

Reply:
So, will we change this to be `Page` or let it be as it is?

Reply:
The page is the main consumer.
But maybe you want to make the status bar red to show something?
Like an error. Or that such an app is doing something.

Reply:
The problem is that this code is not tied to the page. It's global for the entire application.
And when we go to each page, we have to use this code to get the status bar of the desired color.
 Otherwise the style from previous page will remain.

Reply:
@KSemenenko for now we will let the responsibility to the user, if we see a demand we can implement this later